### DISCUSSION TITLE
Interference with laravel-permission?

### DISCUSSION THREAD

**Discussion by dsbilling** [spatie/laravel-markdown#25] [General]
Currently implementing a blog into my website, repo available [here](https://github.com/DanielRTRD/daniel.rtrd.no). And before I installed laravel-markdown I did not have any issues with rendering of views. But noticed this issue now (dashboard when logged in):
![image](https://user-images.githubusercontent.com/9788214/146616641-a4209a88-b8db-433a-8c6f-bc7336c57ff5.png)

Removing laravel-markdown temporary will fix the issue, installing it back in it will cause this issue again. Any suggestions?

**Comment by kaptk2** [comment]
I have opened an issue on the laravel-permission that demonstrates this problem: https://github.com/spatie/laravel-permission/issues/2038 if you want to follow along there.

**Comment by erikn69** [comment]
I have opened a PR for this
https://github.com/spatie/laravel-permission/pull/2039

**Reply by freekmurze** [reply]
Merged and tagged 👍
